lib/modules/manager/gomod/artifacts.ts

```diff
@@ -134,9 +134,9 @@ export async function updateArtifacts({
   const vendorDir = upath.join(goModDir, 'vendor/');
   const vendorModulesFileName = upath.join(vendorDir, 'modules.txt');
```

Comment:
so in your case this file doesn't exist?

Reply:
that's right

Reply:
Is there an alternative file which exists which renovate should have been able to detect?

Reply:
Nope
This use case is for when vendored files aren't committed

Reply:
So your use case is to _add_ vendored files, using Renovate? And then presumably once each repo has added them, this feature isn't really needed?

Reply:
No
I don't want to add them to the commit, I wanna use the vendored files in a post update task

Running go mod vendor in post update is missing the host rules so I need it in the context of the update where those vars are available 

See the related discussion for full details of the use case